docs/public/js/anchor-links.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Add visual anchor links to docs-content
+ * Solution for issue #945 - https://github.com/keystonejs/keystone/issues/945
```

Comment:
referencing github issues doesn't sound like a good idea IMO. The issue should be closed once/if this pull request is merged.